pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -82,10 +82,29 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 			},
 		},
 		{
-			BindAddress:        "192.168.59.103",
+			BindAddress:        []string{"192.168.59.103"},
 			HealthzBindAddress: "",
 			MetricsBindAddress: "127.0.0.1:10249",
-			ClusterCIDR:        "192.168.59.0/24",
+			ClusterCIDR:        []string{"192.168.59.0/24"},
+			UDPIdleTimeout:     metav1.Duration{Duration: 1 * time.Second},
+			ConfigSyncPeriod:   metav1.Duration{Duration: 1 * time.Second},
+			IPTables: kubeproxyconfig.KubeProxyIPTablesConfiguration{
+				MasqueradeAll: true,
+				SyncPeriod:    metav1.Duration{Duration: 5 * time.Second},
+				MinSyncPeriod: metav1.Duration{Duration: 2 * time.Second},
+			},
+			Conntrack: kubeproxyconfig.KubeProxyConntrackConfiguration{
+				MaxPerCore:            pointer.Int32Ptr(1),
+				Min:                   pointer.Int32Ptr(1),
+				TCPEstablishedTimeout: &metav1.Duration{Duration: 5 * time.Second},
+				TCPCloseWaitTimeout:   &metav1.Duration{Duration: 5 * time.Second},
+			},
+		},
+		{
+			BindAddress:        []string{"192.168.59.103", "fd12:3456:789a:1::1"},
+			HealthzBindAddress: "",
+			MetricsBindAddress: "127.0.0.1:10249",
+			ClusterCIDR:        []string{"192.168.59.0/24", "::/0"},
```

Comment:
`::/0`means default route, I guess we should use another CIDR, i.e. "2001:db:1::/64"

Reply:
@aojea @vllry yeah need to pick a CIDR in some unroutable range and use it; perhaps the "site-local" v6 range of FEC0::/10. Just make something up in there.